src/pip/_internal/commands/debug.py

```diff
@@ -43,6 +47,106 @@ def show_sys_implementation():
         show_value('name', implementation_name)
 
 
+def find_vendor_txt_path():
+    # type: () -> str
+    vendor_txt_path = os.path.join(
+        os.path.dirname(pip_location),
+        '_vendor',
+        'vendor.txt'
+    )
+    assert os.path.exists(vendor_txt_path), \
```

Comment:
I thinks this assert can be dropped and `find_vendor_txt_path` inlined in `create_vendor_txt_map`.

Reply:
done